apps/desktop-wallet/src/features/modals/modalTypes.ts

```diff
@@ -16,7 +16,7 @@ You should have received a copy of the GNU Lesser General Public License
 along with the library. If not, see <http://www.gnu.org/licenses/>.
 */
```

Comment:
- [x] Color of WC logo for light theme should be white?

<img width="379" alt="image" src="https://github.com/user-attachments/assets/472cbf63-6434-41b8-9804-938d2b668a01" />

Reply:
It is addressed in my PR.